shared/actions/notifications.js

```diff
@@ -46,13 +45,9 @@ const setupEngineListeners = () => {
   getEngine().setIncomingCallMap({
     'keybase.1.NotifyBadges.badgeState': ({badgeState}) =>
       Saga.put(NotificationsGen.createReceivedBadgeState({badgeState})),
-    'stellar.1.notify.paymentNotification': refreshPayments,
```

Comment:
@cjb i'm going to just mark this as a flowissue so it can be fixed later. also we shouldn't be putting any random stuff into notifications, should go into the correct actions folder (wallets)